omod/src/main/java/org/openmrs/module/registrationapp/RegistrationAppUiUtils.java

```diff
@@ -61,7 +61,23 @@ public String getAttribute(Person person, String attributeTypeUuid) {
 		
 		return null;
 	}
-	
+	/**
+	 * @since 1.15.0-SNAPSHOT
```

Comment:
The since annotation points to the version which will be released with the current snapshot.

Reply:
the PR has been updated to address the above issues